src/test/scala/govuk/Signon.scala

```diff
@@ -27,8 +27,8 @@ object Signon {
     http("Authenticate in signon")
       .post(signonUrl)
       .formParam("authenticity_token", """${signonAuthToken}""")
-      .formParam("user[email]", sys.env.get("USERNAME").get)
-      .formParam("user[password]", sys.env.get("PASSWORD").get)
+      .formParam("user[email]", sys.env.get("GATLING_USERNAME").get)
```

Comment:
I think there is a comment near the top of this file that also references these two env vars

Reply:
Good spot, thanks 😉